deploy/charts/rook-ceph/templates/clusterrole.yaml

```diff
@@ -27,6 +27,10 @@ rules:
   - apiGroups: ["apiextensions.k8s.io"]
     resources: ["customresourcedefinitions"]
     verbs: ["get"]
+  - apiGroups: ["apps"]
+    resources: ["deployments"]
```

Comment:
Was this added for the osd prepare jobs? We can remove it now, right?

Reply:
Yes. This is not needed anymore.